apis/config/Makefile

```
@@ -1,6 +1,10 @@
 # Copyright 2022 VMware, Inc. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-RELATIVE_ROOT=../..
+include ../../common.mk
 
-include $(RELATIVE_ROOT)/common.mk
+RELATIVE_ROOT=.
+
+.PHONY: manifests
+manifests: ## Generate manifests e.g. CRD, RBAC etc.
```

Nit (optional): use American writing style commas for 'e.g.' and 'etc.'

```suggestion
manifests: ## Generate manifests, e.g., CRD, RBAC, etc.
```

---

Thanks for the suggestion. I'll make this change if there are any other comments I need to address.